wp-content/themes/core/routes/single/css/single.pcss

```diff
@@ -0,0 +1,22 @@
+
+.item-single__content {
+	margin-top: var(--spacer-60);
+}
+
+.item-single__featured-image {
+	max-width: var(--grid-10-col);
+	margin: 0 auto;
+}
+
+.item-single__featured-image-caption {
+	margin: 10px var(--grid-margin-small);
+	font-size: var(--font-size-body-xsmall);
+	line-height: 24px;
+	font-family: var(--font-family-sans-serif);
+	border-bottom: 1px solid var(--color-border);
+
+	@media (--viewport-medium) {
+		width: var(--grid-8-col);
+		margin: 10px auto;
+	}
```

Comment:
Same re: existing layout & typography classes. There's a t-caption class for typography.